integrationTests/chainSimulator/bridge/executeAndBurn_test.go

```diff
@@ -72,19 +72,11 @@ func TestChainSimulator_ExecuteWithMintMultipleEsdtsAndBurnNftWithDeposit(t *tes
 		t.Skip("this is not a short test")
 	}
 
-	nft := sovChainPrefix + "-SOVNFT-123456"
-	nftNonce := uint64(3)
 	nftV2 := sovChainPrefix + "-NFTV2-a1b2c3"
 	nftV2Nonce := uint64(10)
 	token := sovChainPrefix + "-TKN-1q2w3e"
 
 	bridgedInTokens := make([]chainSim.ArgsDepositToken, 0)
-	bridgedInTokens = append(bridgedInTokens, chainSim.ArgsDepositToken{
```

Comment:
Why were these removed from these tests?

Reply:
`NonFungible` type is not accepted anymore